src/Uno.UI/UI/Xaml/Controls/ContentDialog/ContentDialog.cs

```diff
@@ -305,10 +315,13 @@ void Complete(ContentDialogButtonClickEventArgs a)
 				if (!a.Cancel)
 				{
 					const ContentDialogResult result = ContentDialogResult.Primary;
-					_tcs.SetResult(result);
 					PrimaryButtonCommand.ExecuteIfPossible(PrimaryButtonCommandParameter);
 
-					Hide(result);
+					if(Hide(result))
+					{
+						Console.WriteLine("tcs primary");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5229032)